README.rst

```diff
@@ -37,10 +37,39 @@ All packages in apptools require:
 
 * `traits <https://github.com/enthought/traits>`_
 
+Certain sub-packages within apptools have their own specific dependencies,
+which are optional for apptools overall.
+
 The `apptools.preferences` package requires:
 
 * `configobj <http://pypi.python.org/pypi/configobj>`_
 
+To install with `apptools.preferences` dependencies::
+
```

Comment:
Thank you for adding these. 

Do you mind pulling this out to a separate installation section? It interleaving with the list of dependencies makes this section difficult to parse at a glance.

Nit: The single backtick does not change the rendering styling. To match the styling in the first section, using bold text, e.g. `**apptools.preferences**`, might be a neater.

Reply:
I ended up leaving the install instructions as code blocks matching the pyface example.  I think with install instructions in a separate section this looks clean, but I am happy to change to the ** ** approach if you prefer it